packages/paste-website/src/component-examples/StatusPatternExamples.tsx

```diff
@@ -0,0 +1,511 @@
+      backgroundColor="colorBackgroundAvailable" />
+    <Text marginLeft="space30">Available</Text>
+  </Box>
+  <Box display="flex" alignItems="center">
+    <Box
+      borderRadius="borderRadiusCircle"
+      size="sizeSquare30"
+      backgroundColor="colorBackgroundBusy" />
+    <Text marginLeft="space30">Busy</Text>
+  </Box>
+  <Box display="flex" alignItems="center">
+    <Box
+      borderRadius="borderRadiusCircle"
+      size="sizeSquare30"
+      backgroundColor="colorBackgroundUnavailable" />
+    <Text marginLeft="space30">Unavailable</Text>
+  </Box>
+  <Box display="flex" alignItems="center">
+    <Box
+    borderRadius="borderRadiusCircle"
+    borderWidth="borderWidth20"
+    borderStyle="solid"
+    borderColor="colorBorder"
+    size="sizeSquare30"
+    />
+    <Text marginLeft="space30">Offline</Text>
+  </Box>
+</Box>
+`.trim();
+
+export const prefixSuffixTooltipExamples = `
+<Stack orientation="vertical" spacing="space60">
+  <Box display="flex">
+    <ProcessErrorIcon
+      color="colorTextIconError"
+      decorative={false}
+      title="failed to process"
+      size="sizeIcon40"
+    />
+    <Text
+      lineHeight="lineHeight40"
+      fontSize="fontSize40"
+      marginLeft="space20"
+    >
+      Failed to process
+    </Text>
+  </Box>
+  <Box display="flex">
+    <Text marginRight="space20">Failed to process</Text>
+    <ProcessErrorIcon
+      color="colorTextIconError"
+      decorative={false}
+      title="failed to process"
+    />
+  </Box>
+</Stack>
+`.trim();
+
+export const tableExample = `
+<>
+  <Heading as="h2" variant="heading20">
+    Exports
+  </Heading>
+  <Table>
+    <THead>
+      <Tr>
+        <Th>Status</Th>
+        <Th>Export name</Th>
+      </Tr>
+    </THead>
+    <TBody>
+      <Tr>
+        <Td>
+          <Box display="inline-flex" columnGap="space20">
+            <ProcessSuccessIcon
+              color="colorTextIconSuccess"
+              decorative={false}
+              title="success"
+            />
+            <Box>
+              <Text as="div">Successfully exported</Text>
+              <Text as="div" color="colorTextWeak">Tuesday, January 2, 2021</Text>
+            </Box>
+          </Box>
+        </Td>
+        <Td>
+          <strong>Contacts who did not click</strong>
+        </Td>
+      </Tr>
+      <Tr>
+        <Td>
+          <Box display="inline-flex" columnGap="space20">
+            <ProcessSuccessIcon
+              color="colorTextIconSuccess"
+              decorative={false}
+              title="success"
+            />
+            <Box>
+              <Text as="div">Successfully exported</Text>
+              <Text as="div" color="colorTextWeak">Tuesday, February 2, 2021</Text>
+            </Box>
+          </Box>
+        </Td>
+        <Td>
+          <strong>All contacts</strong>
+        </Td>
+      </Tr>
+    </TBody>
+  </Table>
+  <Box marginTop="space40">
+    <Heading as="h2" variant="heading20" marginTop="space40">
```

Comment:
```suggestion
    <Heading as="h2" variant="heading20">
```

Reply:
I don't think Heading accepts top margins.

Reply:
you are right! I think I was trying something here and forgot to take it off. Good catch!